crates/astria-composer/src/composer.rs

```diff
@@ -28,6 +32,13 @@ pub struct Composer {
     /// pending transactions from them and wraps them as sequencer transactions
     /// for submission.
     searcher: Searcher,
+    /// The handle to communicate SequenceActions to the Executor
+    /// This is at the Composer level to allow its sharing to various different collectors.
+    executor_handle: ExecutorHandle,
+}
+
+struct ExecutorHandle {
+    send_bundles: Sender<SequenceAction>,
```

Comment:
is this meant to send bundles (ie `Vec<SequenceAction>`) like the name, or it is intended to send individual `SequenceAction`s?

Reply:
Right now, it sends individual bundles. converting it to an array would require batching of sorts in `GethCollector`(or `EthCollector`) which can complicate things for now. We can do it if required.

Reply:
i'd keep the name `serialized_rollup_transactions_tx/rx` as it appears in `searcher`